### PR TITLE
Fixes for Tizen GCC

### PR DESCRIPTION
**Pull request by georged-youi**

### CODE REVIEW

File: CMake/FollyCompilerUnix.cmake [hunk 1]
```diff
@@ -1,6 +1,11 @@
 set(CMAKE_CXX_FLAGS_COMMON "-g -Wall -Wextra")
 set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_COMMON}")
-set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_COMMON} -O3")
+if(TIZEN_NACL)
+  set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_COMMON} -Os")
```
Comment (mathieumallet) [inline CMake/FollyCompilerUnix.cmake:4]: why is this needed?

Comment (georged-youi) [reply CMake/FollyCompilerUnix.cmake:4]: because the compiler is broken. it will be in the final pr:
# gcc is buggy and does not work with -O2/3, as it fails with this error:
# internal compiler error: in set_minipool_offset, at config/arm/arm.c:15988

Comment (georged-youi) [reply CMake/FollyCompilerUnix.cmake:4]: in order to use tizen's gcc, we need to use -Os instead of -O2/3 everywhere

Comment (mathieumallet) [reply CMake/FollyCompilerUnix.cmake:4]: mm Os is a pretty big difference vs O3 in terms of performance

Comment (georged-youi) [reply CMake/FollyCompilerUnix.cmake:4]: well, it is what it is. Os is similar to O2, but with size in mind. this was a known issue from the beginning, but the comparison between gcc hermes and clang nacl executers was significant.

**Review by mathieumallet — APPROVED** [review]
